watcher_jobs.go

```diff
@@ -112,7 +112,7 @@ func (jw *JobsWatcher) refresh(o *Observer, infoKeys []string, rawMetrics map[st
 				continue
 			}
 
-			ch <- prometheus.MustNewConstMetric(pm.desc, pm.valueType, pv, rawMetrics[ikClusterName], rawMetrics[ikService], stats["ns"], stats["set"], stats["module"], stats["job-type"], stats["trid"])
+			ch <- prometheus.MustNewConstMetric(pm.desc, pm.valueType, pv, rawMetrics[ikClusterName], rawMetrics[ikService], stats["ns"], stats["set"], stats["module"], stats["job-type"], stats["trid"], stats["sindex-name"])
```

Comment:
From where are we getting sindex-name ?

Reply:
Well not exactly a breaking change. But the label 'sindex_name' could go empty for older server releases.

Reply:
@spkesan I thought that an empty label would be fine. Do you suggest we only add some logic to only add the label for 6.1+ servers?

Reply:
Oh, you're right. I think we are good here.
https://prometheus.io/docs/concepts/data_model/
> _A label with an empty label value is considered equivalent to a label that does not exist._